templates/zerver/index.html

```diff
@@ -45,14 +45,17 @@
 
 {% endblock %}
 {% block content %}
+{% include "zerver/lightbox_overlay.html" %}
+{% include "zerver/subscriptions.html" %}
+{% include "zerver/drafts.html" %}
+<div id="settings_overlay_container" class="overlay" data-overlay="settings" aria-hidden="true">
+  {% include "zerver/settings_overlay.html" %}
+</div>
+
```

Comment:
Can we put these below `css-loading`?  I think that one belongs at the top.

Reply:
This just moves them out of `.app-main`, right?  That looks pretty safe.

Reply:
Yep, it’s actually safe since I’ve checked the selectors.

Reply:
Yep, it’s actually safe since I’ve checked the selectors.

Reply:
But also yes, that ordering doesn’t matter. `.css-loading` does not have any impact.